pkg/config/config.go

```diff
@@ -74,6 +74,7 @@ func init() {
 	Datadog.SetDefault("confd_dca_path", defaultDCAConfdPath)
 	Datadog.SetDefault("use_service_mapper", true)
 	Datadog.SetDefault("additional_checksd", defaultAdditionalChecksPath)
+	Datadog.SetDefault("log_payload", true)
```

Comment:
let's default to `false` or this is going to be spammy :)

Reply:
Also please don't hate me but can we call this `log_payloads`? 😊

Reply:
That should definitely be false! Good catch. Since I'm already fixing, I can definitely change the name haha.